_posts/en/posts/2016-02-18-release-0-12-0.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Bitcoin Core 0.12.0 Released!
+name: blog-release-0.12.0
+id: en-blog-release-0.12.0
+lang: en
+permalink: /en/2016/02/18/release-0.12.0/
+type: posts
+layout: post
+share: true
+version: 1
+---
+We're very excited to announce the official release of Bitcoin Core v0.12.0. A lot of hard work has gone into this release and it may just be the biggest one yet, with more significant improvements than any other before.
+
+Here are the major improvements you’ll get to benefit from if you upgrade your nodes to version 0.12:
+
+1. Up to 7x Faster Signature Validation
+2. Ability to Limit Upload Traffic
+3. Crash Prevention via Memory Pool Limits
+4. Option to Send Transactions That Can Be Fee-Boosted
```

Comment:
You're talking about RBF right?  There isn't support for sending RBF txs, only for relaying them.